### PR TITLE
[n/a] remove old css and set up for new css stuff

### PR DESCRIPTION
**Pull request by leobauza**
Didn't want to just merge this in:

@efatsi making sure this all looks ok to you

@peytonchance if you have css written lets sync up to get it into this new way of doing things...also let's chat about how you are going to tackle this. I set up a reset and some folder structure that might be helpful.

### CODE REVIEW

File: app/javascript/css/base/vars.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  --width-full: 100%;
+  --width-half: 50%;
+  --width-quarter: 25%;
+  --width-416: 26rem;
+  /* border-radius */
+  --rounded: 20px;
+  --rounded-full: 9999px;
+  /* shadows */
+  --shadow: 0px 6px 6px 4px rgba(0, 0, 0, 0.25);
+}
```
Comment (leobauza) [inline app/javascript/css/base/vars.css:31]: These can all go, or all change, or whatever...just dropped the a sample set of vars we use for the new FED quiz that can be adjusted

File: app/javascript/css/components/sample.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  border: 1px dashed #d67c7c;
+}
+
+.round-active {
+  border: 1px dashed #d67c7c;
+}
+
+.round-done {
+  border: 1px dashed #c6b9b9;
+}
```
Comment (leobauza) [inline app/javascript/css/components/sample.css:99]: Assuming all this CSS will go away I just grabbed all the css that existed so far

**Review by peytonchance — APPROVED** [review]
👍

File: app/assets/stylesheets/application.css [hunk 3]
```diff
@@ -10,16 +10,5 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require darkmode
```
Comment (efatsi) [inline app/assets/stylesheets/application.css:13]: RIP

**Pull request merged**